ui/app/routes/vault/cluster/access/mfa/methods/method.js

```diff
@@ -1,8 +1,22 @@
 import Route from '@ember/routing/route';
-
+import { hash } from 'rsvp';
```

Comment:
Jordan might have some thoughts on Hash. I think there's a work around he mentioned, but I know we're under a time crunch so this is more of a if you're interested reach out to him about it.